python/ray/dashboard/modules/job/tests/test_job_manager.py

```diff
@@ -174,6 +174,40 @@ async def test_get_all_job_info(call_ray_start, tmp_path):  # noqa: F811
     assert found
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "call_ray_start",
+    ["ray start --head"],
+    indirect=True,
+)
+async def test_delete_job_info(call_ray_start, tmp_path):  # noqa: F811
```

Comment:
can you add an e2e test with dashboard endpoint as well? just call /delete_job and make sure both job id and submission id is deleted

Reply:
Done.